index/scorch/merge.go

```diff
@@ -82,7 +83,36 @@ OUTER:
 	s.asyncTasks.Done()
 }
 
-func (s *Scorch) planMergeAtSnapshot(ourSnapshot *IndexSnapshot) error {
+func (s *Scorch) parseMergePlannerOptions() *mergeplan.MergePlanOptions {
+	mergePlannerOptions := &mergeplan.DefaultMergePlanOptions
```

Comment:
oh, this seems it'll have the effect of inadvertently updating the default merge plan options, when instead it should be parsing into its own copy?

Reply:
Whoops..fixed!